### ISSUE TITLE
Compass View should display attribute (icons)

### ISSUE THREAD

**Issue by bmuzzin** [cgeo/cgeo#12744]
### Tell us your idea!

I use the compass rose view almost exclusively when nearing a cache. If I don't find said cache within a few minutes, I usually check the attributes and hint. The hint is easily displayed in the compass view with the button, but to get the attributes, I need to back out of the compass view, and then go into "More Details".

### What solution would you suggest?

Display the attribute icons on the screen (eg. either at the top or bottom) of the compass screen.

### Is there an alternative?

You can back out of the compass screen, and go into "More details" to find the attributes.  However, after I do this, I usually immediately go back to the compass view.

### Additional Information

Suggest location of attribute icons, up by other cache info.

![Screenshots_2022-02-14-17-00-35](https://user-images.githubusercontent.com/3506490/153953831-85e02ea7-795c-491a-b84b-6d9f1b6646e0.png)

**Comment by Lineflyer** [comment]
Interesting idea I never thought about but find very interesting.

I would suggest however to rather show them below the compass rose to avoid the rose pushed out of view on smaller screen devices.
Secondly I guess not all attributes are helpful in such scenario. Now that we have a grouping of attributes we could try to identify what group(s) could be shown here.

**Comment by bmuzzin** [comment]
Certainly, where I'm from and given the time of year, available (or not) in winter is important 😁. I've got to imagine most are useful, for different cachers and the situation. Since a cache can only have a certain number (10?), it doesn't seem like it should take up too much real estate to have all of them?

**Comment by ztNFny** [comment]
> Since a cache can only have a certain number (10?), it doesn't seem like it should take up too much real estate to have all of them?

There's no limit to the number of attributes (other than the number of available attributes, which is ~ 80)

**Comment by Lineflyer** [comment]
There is (for gc.com acc. to their hiding page):
![image](https://user-images.githubusercontent.com/949669/154132613-1c3d02b2-8ec8-4521-b4b6-3e4ed9b1e038.png)

**Comment by ztNFny** [comment]
> There is (for gc.com acc. to their hiding page):

Either not enforced or recent, e.g. https://coord.info/GC4CF4B has more
